styles.css

```diff
@@ -0,0 +1,160 @@
+:root {
+  background-color: green;
+}
+h1 {
+  margin-top: 20px;
+  color: white;
+}
+body {
+  background-color: green;
+  width: 100%;
+  display: block;
+  align-items: center;
+  justify-content: space-evenly;
+  text-align: center;
+}
+.card {
+  display: inline-block;
+  position: relative;
+  width: 2.5em;
+  height: 3.5em;
+  background-color: #ffffff;
+  /* box-shadow: 0 0.0625em 0.125em rgba(0, 0, 0, 0.15); */
+  border-radius: 0.5em;
+  margin-left: 10px;
+  transition: transform 1s;
+  transform-style: preserve-3d;
+  cursor: pointer;
+}
+.card-topleft {
+  /* position: absolute; */
+  top: 0.25em;
+  left: 0.55em;
+}
+.card-corner-rank,
+.card-corner-suit {
+  width: 1em;
+  text-align: center;
+  transform: translate(-50%, 0);
+}
+.suit {
+  margin: 5px;
+  font-size: 30px;
+  width: 1em;
+  transform: translate(-50%, 0);
+}
+
+.red {
+  color: red;
+}
+.cardContainer {
+  background-image: url("Assets/table.png");
```

Comment:
Consider using newlines to group your CSS properties and comments for easier understanding of what you are trying to achieve.